### PR TITLE
update oyenolaphilipinc.json

### PR DESCRIPTION
**Pull request by oyenolaphilipinc**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4585"></a>

### CODE REVIEW

File: data/oyenolaphilipinc.json [hunk 1]
```diff
@@ -23,4 +23,23 @@
     "DevOps",
     "DevRel"
   ]
```
Comment (priyankarpal) [inline data/oyenolaphilipinc.json:25]: Please add a `,` after bracket
``` suggestion
  ],
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/oyenolaphilipinc.json (hunk 1 above)
Comment (oyenolaphilipinc) [reply data/oyenolaphilipinc.json]: already added it and fixed it

**Review by priyankarpal — APPROVED** [review]
Yeah! 😎 Looks great.

**Review by Hrishi5111998 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**